test/system/posts_test.rb

```diff
@@ -0,0 +1,40 @@
+require 'application_system_test_case'
+
+class PostsTest < ApplicationSystemTestCase
+  setup do
+    @userone = users(:one)
+    @usertwo = users(:two)
+    @postone = posts(:pone)
+    # @posttwo = posts(:ptwo)
+  end
+  test 'Post index page' do
+    visit user_posts_path(@userone)
+    assert 'avator.png', @userone.photo
+    assert_selector 'a', text: @userone.name
+    assert_selector 'p', text: "Number of post #{@userone.posts_counter}"
+
+    @userone.posts.each do |post|
+      assert_selector 'a', text: post.title
+      assert_selector 'p', text: post.text
+      assert_selector 'span', text: post.comments.first.text
+      assert_selector 'span', text: post.comments_counter
+      assert_selector 'span', text: post.likes_counter
+    end
+    assert_selector 'a', text: 'Pagination'
+    click_link @userone.posts.first.title, match: :first
+    assert user_post_path(@userone, @userone.posts.first)
+  end
```

Comment:
- [ ] Please make sure that you have a Spec file that only tests the `Post` index page :pray: